flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/dataformat/vector/writable/WritableBytesVector.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.dataformat.vector.writable;
+
+import org.apache.flink.table.dataformat.vector.BytesColumnVector;
+
+/**
+ * Writable {@link BytesColumnVector}.
+ */
+public interface WritableBytesVector extends WritableColumnVector, BytesColumnVector {
+
+	/**
+	 * Set byte[] at rowId with the provided value.
+	 */
+	void appendBytes(int rowId, byte[] value, int offset, int length);
```

Comment:
Rename to `setBytes`?

Reply:
I'll add comment.
`Note: Must append values according to the order of rowId, can not random append.`